packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -156,14 +170,19 @@ const styles = (theme: ThemeType) => ({
 interface DialogueMatchRowProps {
   rowProps: DialogueUserRowProps<boolean>; 
   classes: ClassesType<typeof styles>; 
-  showMatchNote: boolean;
   onHide: ({ dialogueCheckId, targetUserId }: { dialogueCheckId: string|undefined; targetUserId: string; }) => void;
 }
 
-const DialogueMatchRow = ({ rowProps, classes, showMatchNote, onHide }: DialogueMatchRowProps) => {
-  const { DialogueCheckBox, UsersName, MessageButton, DialogueNextStepsButton, PostsItem2MetaInfo } = Components
+const DialogueMatchRow = ({ rowProps, classes, onHide }: DialogueMatchRowProps) => {
+  const { DialogueCheckBox, UsersName, MessageButton, DialogueNextStepsButton, PostsItem2MetaInfo, ReactionIcon } = Components
 
   const { targetUser, checkId, userIsChecked, userIsMatched } = rowProps;
+  const { document: dialogueCheck } = useSingle({
+    fragmentName: "DialogueCheckInfo",
+    collectionName: "DialogueChecks",
+    documentId: checkId,
+  });
```

Comment:
Don't we already have the dialogue checks in the parent (wherever all the rows are)?  I don't think we need to fetch it again; we can just pass in the entire check object to `DialogueMatchRow` rather than just the check id.  Would really prefer to not make an additional x00 queries on the dialogue matching page.